js/bundle.go

```diff
@@ -132,7 +137,13 @@ func NewBundleFromArchive(
 	}
 
 	c := compiler.New(logger)
-	pgm, _, err := c.Compile(string(arc.Data), arc.FilenameURL.String(), "", "", true, compatMode)
+	c.COpts = compiler.Options{
+		Strict:            true,
+		CompatibilityMode: compatMode,
+		SourceMapEnabled:  true,
+		SourceMapLoader:   generateSourceMapLoader(logger, arc.Filesystems),
+	}
+	pgm, _, err := c.Compile(string(arc.Data), arc.FilenameURL.String(), true, c.COpts)
```

Comment:
This can probably be reverted, but still seems kind of nicer to me so I am leaving it for now

Reply:
yeah, I mean that currently, this doesn't let you wrap the code in w/e you want. If it's dependency it will wrap it in the `function(module, exports ...` wrapper if not it won't. And all of that with 1 boolean instead of 2 strings. 

This was never used for anything apart from that outside of tests so :shrug: it seems like a good change to me